src/__tests__/homecomponent.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import ReactDOM from 'react-dom';
+import Mockedhome from '../mocks/mockedhomecomponent';
+import '@testing-library/jest-dom';
+
+describe('Test Home page Components', () => {
+  it('test HomePage function component renders without error', () => {
+    const divElement = document.createElement('div');
+    ReactDOM.render(<Mockedhome />, divElement);
+  });
+
+  it('render correctly', () => {
+    const { getByTestId } = render(<Mockedhome />);
```

Comment:
- [ ] You simply created a mock component here, rather you should test actual components used in the project, kindly import actual components used in the project and pass mock props to them then test them.